deepchecks/tabular/checks/data_integrity/data_duplicates.py

```diff
@@ -118,3 +122,30 @@ def run_logic(self, context: Context, dataset_kind):
             display = None
 
         return CheckResult(value=percent_duplicate, display=display)
+
+    @docstrings
+    @merge_args(SingleDatasetCheck.run)
```

Comment:
following our discussion in algo slack, Would make it possible to supply check result to the fix function and use it without supplying all of the run mandatory arguments. 
Also would add more text explaining that if no check result is not supplied the fix will rerun the check and if check result is supplied it does not rerun